x-pack/plugins/lens/public/editor_frame_service/editor_frame/state_helpers.ts

```diff
@@ -40,6 +40,7 @@ import {
 import type { DatasourceStates, DataViewsState, VisualizationState } from '../../state_management';
 import { readFromStorage } from '../../settings_storage';
 import { loadIndexPatternRefs, loadIndexPatterns } from '../../data_views_service/loader';
+import { isAnnotationsLayer } from '../../visualizations/xy/visualization_helpers';
```

Comment:
nit: Could we move this helper into the `common/convert_to_lens` directory so the editor frame doesn't have a direct dependency on the xy vis? Two reasons:
* This rule makes coupling between the frame and specific chart types more explicit as it has to go through a central place (the `common` directory)
* Harder to run into cyclic imports

Reply:
Done!